modules/webgl/src/classes/texture.js

```diff
@@ -696,7 +696,15 @@ export default class Texture extends Resource {
   }
 
   _isNPOT() {
-    return !isWebGL2(this.gl) && (!isPowerOfTwo(this.width) || !isPowerOfTwo(this.height));
+    if (isWebGL2(this.gl)) {
+      // NPOT restriction is only for WebGL1
+      return false;
+    }
+    // Width and height not available, consider it is not NPOT texture
+    if (!this.width || !this.height) {
```

Comment:
Interesting... what was happening that required this.

Reply:
texture-cube initialization is not properly setup. #1070